src/components/addToCart/AddToCard.tsx

```diff
@@ -0,0 +1,68 @@
+import classNames from 'classnames';
+import { useContext, useEffect, useState } from 'react';
+import { setLocalStorageItem } from '../../helpers/util';
+import { addBasket, GlobalContext, removeProductInBasket } from '../../reducer';
+import { Product } from '../../types/product';
+import { ShoppingProduct } from '../../types/shoppingProduct';
+
+import './addToCart.scss';
+
+type Props = {
+  product: Product;
+};
+
+export const AddToCart: React.FC<Props> = ({ product }) => {
+  const [state, dispatch] = useContext(GlobalContext);
+  const [selected, setSelected] = useState(false);
```

Comment:
Boolean variables should be prefixed with 'is' following to Naming Conventions

Reply:
+